cmd/registration-dns/regforwarder.go

```diff
@@ -0,0 +1,138 @@
+package main
+
+import (
+	"bytes"
+	"io/ioutil"
+	"log"
+	"net/http"
+
+	"github.com/refraction-networking/gotapdance/pkg/dns-registrar/responder"
+	pb "github.com/refraction-networking/gotapdance/protobuf"
+	"google.golang.org/protobuf/proto"
+)
+
+// Interface to forward DNS registration requests. Use a dns responder to receive requests and send responses.
```

Comment:
golang doc comments begin with the name of the function or the object:

```go
// DnsRegForwarder provides an interface to forward DNS registration requests. Use a dns ...
type DnsRegForwarder struct {
```

Just noting this once here, I do see the correct format in some other places. Just want to make sure we are consistent and following proper doc syntax

Reply:
I did not know that! Thanks for pointing that out, I've changed the comments to follow the proper syntax.